src/domain/discounts/new/discount-form/edit-conditions-modal.tsx

```diff
@@ -28,7 +28,7 @@ const EditConditionsModal: React.FC<EditConditionsModalProps> = ({
   )
 }
 
-const getTitle = (view: DiscountConditionType) => {
+export const getTitle = (view: DiscountConditionType) => {
```

Comment:
very nit: maybe create a new folder `/utils` in the discount domain and move this there instead of exporting it from here